pkg/neotest/coverage.go

```diff
@@ -201,6 +202,27 @@ func processCover() map[documentName][]*coverBlock {
 		for _, b := range mappedBlocks {
 			blocks = append(blocks, b)
 		}
+		slices.SortFunc(blocks, func(a, b *coverBlock) int {
+			if a.startLine != b.startLine {
+				return int(a.startLine - b.startLine)
```

Comment:
uint to int... Maybe we can just `cmp.Or(cmp.Compare(), cmp.Compare()...)`?

Reply:
Sure we can! It's just because I have no experience in updating NeoGo to Go 1.22.